test/unit/iPgmUnit.js

```diff
@@ -133,6 +133,58 @@ describe('iPgm Class Unit Tests', () => {
 
       expect(pgm.toXML()).to.equal(expectedXML);
     });
+
+    it('regular <parm> contains by=\'val\'', () => {
```

Comment:
These tests should also be run through the linter (looks like the big issue is just spacing and use of single-quotes instead of double-quotes)